docs/etiquette/data-monitoring.md

```diff
@@ -0,0 +1,222 @@
+
+### Customize
+Most customization required for the hallMonitor script is handled by the setup command above. However, if REDCap column names need to be re-mapped to match the expectations of the instruments script, this can be built directly into hallMonitor. There are two options available:
+
+#### one-to-one mapping
+If just the instrument name requires remapping, the `-m` flag allows you to specify this. Here is an example where the mistakenly named "sas" instrument is mapped to "sas2":
+![hallmonitor-m-flag](https://raw.githubusercontent.com/NDCLab/wiki/main/docs/_assets/hpc/hallmonitor-m-flag.png)
+
+#### header replacement
+If both the instrument name and the item numbers require remapping, you will use the `-r` flag to replace all column names. You will need to create a comma-separated list of the column names that should be used:
+1. Right-click the REDCap .csv file and open with a text editor.
+2. Copy the entire first line (careful: this first line may span _visually_ over several lines in the text editor) except for "record_id" onto your clipboard.
+3. Insert into hallMonitor.sub following the instructions below.
+
+To modify hallMonitor.sub:
+1. Navigate to the data-monitoring folder: `cd your-dataset/data-monitoring`.
+2. Open the .sub file: `nano hallMonitor.sub`.
+3. Arrow down to the last line (`./hallMonitor.sh`), add a space, then the flag (`-m` or `-r`) and the mapping or replacement details.
+4. Save your changes by holding down the ctrl key and pressing the letter 'o'. (Note: this is ctrl on both PC and Mac, not cmd.) You will be prompted to save the file, which you do by hitting enter/return.
+5. To exit the nano view, hold down the ctrl key again and press the letter 'x'.
+
+If you have added any modification to the REDCap header, this is executed (by running hallMonitor.sub) via the rename-cols.py script.
+
+#### update-tracker.py
+You need to inform hallMonitor which columns it should use to check for the existence of data in the REDCap .csv so that it can update the instrument-specific data points in your central tracker. In general, this is best accomplished with the "timestamp" column in REDCap, which only has data if a participant completed the specific questionnaire. To connect your REDCap data to your central tracker, complete the dictionary in update-tracker.py appropriately. Use the same `nano` command as above. Here is an example where hallMonitor will look for the existence of data for demo_b and bfne_b based on their timestamp:
+```
+redcheck_columns = {
+    "demo_b_s1_r1_e1_timestamp" : "demo_b_s1_r1_e1", 
+    "bfne_b_s1_r1_e1_timestamp" : "bfne_b_s1_r1_e1"}
+```
+
+### How to Run
+The hallMonitor script should be run weekly by the project lead. It is recommended that you create a recurring reminder on your calendar or on Slack (`/remind me on fridays 'run hallMonitor'`).
```

Comment:
There's also an option to create automatic and scheduled runs of scripts. Should I insert a link to a tutorial here?